CHANGELOG.md

```diff
@@ -60,6 +60,8 @@ Bug fixes:
 - Fix `GeoDataFrame.merge()` incorrectly returning a `DataFrame` instead of a
   `GeoDataFrame` when the `suffixes` argument is applied to the active
   geometry column (#2933).
+- Fix `to_postgis()` method to make it compatible with psycopg3 as well as psycopg2 (#3167).
```

Comment:
Don't overly mind, but I'd this could be grouped as a new feature, as it's new functionality (I know the original issue is tagged as a bug though)

Reply:
It is more a compatibility note imho.

Reply:
Thanks! I've moved it to compatibility in a0e90cc